draft-ietf-httpbis-proxy-status.md

```diff
@@ -74,11 +76,78 @@ This specification uses Structured Headers {{!I-D.ietf-httpbis-header-structure}
 Note that in this specification, "proxy" is used to indicate both forward and reverse proxies, otherwise known as gateways. "Next hop" indicates the connection in the direction leading to the origin server for the request.
 
 
-# The Proxy-Status HTTP Header Field {#header}
+# The Proxy-Info HTTP Header Field {#info-header}
+
+The Proxy-Info HTTP response header field allows an intermediary to append debugging information to the HTTP messages that it sends; it is not intended for use by protocol mechanisms (e.g., caching, content handling, redirection).
+
+Proxy-Info is a Structured Headers {{!I-D.ietf-httpbis-header-structure}} Parameterised List, where each item in the list indicates an intermediary node that has handled the response. The order of the list is in the order the response was handled (i.e., in the direction of origin to user agent).
+
+Each param-item's primary-id is a sh-string that identifies the intermediary generating that value; it MAY be a hostname, IP address, or alias. Its parameters carry Proxy Information Types that convey additional information about that intermediary's handling of the request.
+
+For example:
+
+~~~ example
+HTTP/1.1 200 OK
+Proxy-Info: "ReverseProxy"; f_proto="h1",
```

Comment:
I really don't like the fact that the identifier is now the primary-id, and not `"node"` parameter  like in `Proxy-Status` and `Cache`.